### PR TITLE
Resolve #22

### LINKED ISSUE #22: Disable timeout
Hi,

Normally setting the Lab's timeout to 0 disable it. Here it just leave its default value (2000ms). Is it possible to have the same behavior as Lab?

Thanks

### PR DESCRIPTION
**Pull request by jlouazel**

### CODE REVIEW

**Comment by jlouazel** [comment]
@wtcross: Do you know how to get 100% coverage on this? I can't find any documentation for if/else statements coverage.

File: tasks/lab.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ module.exports = function (grunt) {
 		_.forIn(config, function (configValue, configName) {
 			var option = _.first(_.where(labOptions, { name : configName }));
 
-			if (option && configValue) {
+			if (option && (configValue || typeof configValue === "number")) {
```
Comment (wtcross) [inline tasks/lab.js]: Using the lodash [`_.isNumber`](https://lodash.com/docs#isNumber) function would be ideal. It's the same thing, but lodash is used everywhere else.

**Comment by wtcross** [comment]
There needs to be a test added for the case where the config value is a number. This will bring coverage to 100%.

**Comment by wtcross** [comment]
After looking at the test it appears that this should already be covered...weird.

**Comment by wtcross** [comment]
[This line](https://github.com/wtcross/grunt-lab/blob/master/test/lab_spec.js#L86) is a number!

**Comment by wtcross** [comment]
[This test](https://github.com/wtcross/grunt-lab/blob/master/test/lab_spec.js#L80) is what needs to be corrected. It the task options should look like this:
```javascript
{
  coverage             : true,
  color                : true,
  parallel             : false,
  disableLeakDetection : true,
  reporter             : "console",
  minCoverage          : 100,
  timeout              : 0
}
```

This will require updating the expectations [here](https://github.com/wtcross/grunt-lab/blob/master/test/lab_spec.js#L119) and [here](https://github.com/wtcross/grunt-lab/blob/master/test/lab_spec.js#L150).

**Comment by wtcross** [comment]
After making this change could you squash all your commits into one?

**Comment by wtcross** [comment]
@jlouazel Thank you for catching this and taking the time to fix it. I appreciate it!

**Comment by jlouazel** [comment]
It's fixed. I'm trying to squash all my commits together now.

**Comment by jlouazel** [comment]
This is done captain. You can merge it.

**Comment by wtcross** [comment]
Looks great! Thank you @jlouazel :+1:

**Pull request merged**